core/store/LevelDBStore/LevelDBStore.go

```diff
@@ -18,6 +19,7 @@ func NewLevelDBStore(file string) (*LevelDBStore, error) {
 	// default Options
 	o := opt.Options{
 		NoSync: false,
+		Filter: filter.NewBloomFilter(10),
```

Comment:
Explaining the meaning of bitsPerKey with const name instead of using magic number